themes-default/slim/src/components/quality-pill.vue

```diff
@@ -0,0 +1,292 @@
+            // Check if all resolutions are 1080p
+            // } else if (set(allowed_qualities).issubset(set_1080p)and set(preferred_qualities).issubset(set_1080p)) {
+                // cssClass = qualityCssClassStrings[Quality.FULLHDBLURAY];
+                // text = '1080p';
+            // Check if all resolutions are 720p
+            // } else if (set(allowed_qualities).issubset(set_720p)and set(preferred_qualities).issubset(set_720p)) {
+                // cssClass = qualityCssClassStrings[Quality.HDBLURAY];
+                // text = '720p';
+            // Check if all resolutions are 4K UHD
+            // } else if (set(allowed_qualities).issubset(set_uhd_4k)and set(preferred_qualities).issubset(set_uhd_4k)) {
+                // cssClass = qualityCssClassStrings[Quality.HDBLURAY];
+                // text = '4K-UHD';
+            // Check if all resolutions are 8K UHD
+            // } else if (set(allowed_qualities).issubset(set_uhd_8k)and set(preferred_qualities).issubset(set_uhd_8k)) {
+                // cssClass = qualityCssClassStrings[Quality.HDBLURAY];
+                // text = '8K-UHD';
+            } else {
+                cssClass = 'Custom';
+                text = 'Custom';
+            }
+
+            // @FIXME: Comments
+            /* eslint-enable capitalized-comments */
+
+            return {
+                class: ['quality', cssClass],
+                text
+            };
+        }
+    },
+    methods: {
+        /**
+         * Split a combined quality to allowed and preferred qualities.
+         * Converted Python method from `medusa.common.Quality.split_quality`.
+         * @param {number} quality - The combined quality to split
+         * @returns {Quality} - The split quality
+         */
+        splitQuality(quality) {
+            const { qualityValues } = this;
+            return Object.values(qualityValues).reduce((result, curQuality) => {
+                curQuality = Number(curQuality) >>> 0; // Unsigned int
+                if (curQuality & quality) {
+                    result.allowed.push(curQuality);
+                }
+                if ((curQuality << 16) & quality) {
+                    result.preferred.push(curQuality);
+                }
+                return result;
+            }, { allowed: [], preferred: [] });
+        },
+        /**
+         * Check if all the items of `set1` are items of `set2`.
+         * Assumption: Each array contains unique items only.
+         * Source: https://stackoverflow.com/a/48211214/7597273
+         * @param {(number[]|string[])} set1 - Array to be compared against `set2`.
+         * @param {(number[]|string[])} set2 - Array to compare `set1` against.
+         * @returns {boolean} - Whether or not `set1` is a subset of `set2`
+         */
+        isSubsetOf(set1, set2) {
+            return set1.every(value => set2.includes(value));
+        }
+    }
+};
+</script>
+
+<style scoped>
+span.quality {
```

Comment:
Since you're using scoped and only a single element you can drop the `span` for all the styles.